docs/2. Product Features/08. HSM Templates.md

```diff
@@ -1,18 +1,36 @@
-**How to create HSM templates in Glific and use them in communication?** _4 Minutes Read_
+> ### **6 minutes read         _____________           `Beginner`**
```

Comment:
Did you intentionally added this underscore
<img width="572" alt="image" src="https://github.com/glific/docs/assets/32592458/bb3f16d0-83e0-47ab-bfda-259a251a27fc">

Reply:
Underscore was added to create space, is there any alternative?